README.md

```diff
@@ -1,2 +1,32 @@
-# Be-Secure
-Be-Secure is an umbrella project of open source security tools ,projects and applications.
+<h2 align="center">Be-Secure</h2>
+
+
+# Be-Secure 
+
+Be-Secure is an umbrella of open source security projects and utilities driven as a community initiative by Wipro. The Be-Secure projects are focused on enabling open source community to enhance security for their open source projects and help build a security community around open source. These security projects and utilities will be tracked by Wipro and its network of trusted security partners.  There is a growing trend of open source adoption today and this growth is driving creation of lot more open source components. It is essential to have a mechanism and community to focus on security of these open source software components.
+
+
+There are lot many open source tech stacks out there in the world. As a start, the focus is currently set on the most popular open source tech stacks. These tech stacks have been grouped into 5 different open source security tech stack categories. This approach will help to focus and drive security projects/ utilities more efficiently by addressing the specific security requirements of these tech stacks.
+Be-Secure projects will revolve around a set of predefined open source security tech stacks that are curated based on their usage. 
+
+#### The different open source security tech stacks that has been identified based on the characteristics and type of usage are –
+
+    *   Security of DevOps Tool stack
+    
+    *   Language and framework security stack
+    
+    *   Application security stack 
+    
+    *   Distributed application / blockchain security stack
+    
+    *   Open source security tool security stack
+  
+
+The vision of Be-Secure community is to be one stop for all users to contribute and consume security projects to address security needs of open source projects.
+
+# Be-Secure Projects / Utilities:
+   One of the first utilities to be created as part of Be-Secure project is BeSman.
+
+   •	BeSman – is a command-line utility to provision customized environments for each open source tech stack. These environments are known as BeSman environments. Please refer the following link (https://github.com/Be-Secure/BeSman) to know more about BeSman. It is built using bash and can be easily customized to provision new environments for an open source tech stack based on open source project requirements. Anyone can leverage BeSman utility to build customized environments for their projects.
+  
+For more information on Be-Secure, please visit our [site](https://jobyko.github.io/Be-Secure_web/).
```

Comment:
If the contents have been pulled into the parent repo, please use the GitHub page of the same.

Reply:
@asa1997 
Made the suggested changes.